package.json

```diff
@@ -10,38 +10,37 @@
+    "prestart": "npm run build:core:esm",
     "storybook:build": "build-storybook -o public",
     "prestorybook:build": "npm run build:core:esm",
-    "preintegrate": "shx rm -rf packages/*/src/integrated",
-    "integrate": "concurrently yarn:integrate:*",
-    "preintegrate:core:tokens": "shx mkdir -p packages/core/src/integrated/tokens",
-    "integrate:core:tokens": "shx cp -R node_modules/@onfido/castor-tokens/src/* packages/core/src/integrated/tokens",
     "prebuild": "shx rm -rf packages/*/dist",
     "build": "concurrently yarn:build:*",
     "build:core:css": "sass packages/core/style:packages/core/dist --no-source-map -s compressed",
     "postbuild:core:css": "shx mkdir -p packages/core/dist/themes && shx cp -R node_modules/@onfido/castor-tokens/dist/*.css packages/core/dist/themes",
     "build:core:esm": "tsc -p packages/core/tsconfig.json",
     "build:react:esm": "tsc -p packages/react/tsconfig.json",
     "preprototype": "shx rm -f packages/*/prototype.js*",
-    "prototype": "concurrently yarn:prototype:*",
-    "prototype:react": "esbuild packages/react/src/index.ts --outfile=packages/react/prototype.js --bundle --define:process.env.NODE_ENV=\\\"production\\\" --target=es2015 --sourcemap --global-name=Castor",
+    "prototype": "esbuild packages/react/src/index.ts --outfile=packages/react/prototype.js --bundle --define:process.env.NODE_ENV=\\\"production\\\" --target=es2015 --sourcemap --global-name=Castor",
     "lint": "concurrently yarn:lint:*",
     "lint:commit": "commitlint --from origin/main --to HEAD --verbose",
     "lint:es": "eslint . --ignore-path .gitignore --max-warnings 0",
     "lint:ts": "tsc --noEmit",
     "lint:style": "stylelint '**/*.{,s}css' --ignore-path .gitignore --max-warnings 0",
     "lint:prettier": "prettier . --check --ignore-path .gitignore",
+    "preinspect": "yarn build && yarn prototype",
+    "inspect": "bundlewatch --config .bundlewatchrc.json",
     "test": "jest --coverage",
+    "ui": "concurrently yarn:e2e:serve yarn:ui:open",
+    "ui:open": "yarn cypress open --env failOnSnapshotDiff=false",
     "e2e": "concurrently -k --success first yarn:e2e:*",
     "poste2e": "nyc report -r text -r text-summary",
-    "e2e:run": "CYPRESS_baseUrl=http://host.docker.internal:6006 docker-compose up --abort-on-container-exit --exit-code-from cypress",
+    "e2e:run": "wait-on http://localhost:6006 && yarn cypress run -c video=true",
     "e2e:serve": "NODE_ENV=e2e yarn start --ci --quiet",
-    "snapshot": "CYPRESS_updateSnapshots=true yarn e2e",
-    "inspect": "concurrently yarn:inspect:*",
-    "preinspect:bundle-size": "yarn build && yarn prototype",
-    "inspect:bundle-size": "bundlewatch --config .bundlewatchrc.json",
+    "screenshot": "CYPRESS_updateSnapshots=true yarn e2e",
+    "baseline": "FOLDER=e2e/.screenshots; shx rm -rf $FOLDER &&  gh run download -n screenshots -D $FOLDER $(gh run list -w test | grep $(git branch --show-current) | sed 's/.*\\([0-9]\\{10\\}\\).*/\\1/g')",
```

Comment:
Hate how bad this script reads but only alternatives are either multiple `package.json` scripts, or a custom file with either a Node or Bash script, of which I like neither :/